src/sentry/static/sentry/app/views/organizationEventsV2/transactionView/dividerHandlerManager.tsx

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
+
+import {
+  rectOfContent,
+  clamp,
+  toPercent,
+  UserSelectValues,
+  setBodyUserSelect,
+} from './utils';
+
+// divider handle is positioned at 50% width from the left-hand side
+const DEFAULT_DIVIDER_POSITION = 0.5;
+
+const selectDividerLines = (transform: (dividerDOM: HTMLDivElement) => void): void => {
+  document
+    .querySelectorAll<HTMLDivElement>("[data-test-id='divider-line']")
```

Comment:
`data-test-id` prefix should only be used in tests and was mentioned previously that it might one day be stripped as part of the build process

Reply:
Also, why do we need to use querySelectorAll to select the element? Can't we pass the refs or something like that?

Reply:
@lynnagara  I refactored to make use of refs here https://github.com/getsentry/sentry/pull/14225/commits/5a8bb7fcb27c0caa4196539405b578cf66b1b42d